sms_twilio.js

```diff
@@ -14,6 +14,59 @@ sms.configure = function (options) {
   sms.from = options.from;
 };
 
+var lookups = new Meteor.Collection('meteor_accounts_sms_lookups');
+
+// Check if a number is a mobile number with twilio lookups.
+var isMobileNumber = function (phone) {
+  if (phone.indexOf('+1') === -1) phone = '+1' + phone;
+
+  var lookup = lookups.findOne({ phone_number: phone });
+
+  if (lookup) return lookup.carrier.type === 'mobile';
+
+  // XXX Use twilio lookups node library
+  var response = HTTP.call('GET', 'https://lookups.twilio.com/v1/PhoneNumbers/' + phone, {
+    auth: Meteor.settings.TWILIO.SID + ':' + Meteor.settings.TWILIO.TOKEN,
+    params: {
+      Type: 'carrier'
+    }
+  });
+
+  if (response.data) {
+    lookups.insert(_.extend(response.data, { timestamp: new Date() }));
```

Comment:
don't think we should add the `+1` logic in this package - would hinder use of non US phone nr.?